content/1_docs/2_cookbook/9_setup/0_kirby-in-a-nutshell/recipe.txt

```diff
@@ -0,0 +1,318 @@
+
+(screenshot: panel-dashboard.png)
+
+The form fields and sections for the dashboard are defined in the `/site/blueprints/site.yml` blueprint. The text file for the `$site` object is called `site.txt` and lives directly in the content folder. Next to the `site.txt` file you can store media files that belong to the `$site` object.
+
+## Multi-section pages (type one-pager)
+
+One-pager websites or generally pages that contain different sections (often with repeating types of content) can be realized in multiple ways:
+
+- The sections are created as subpages of the parent page, like in our (link: docs/cookbook/templating/one-pager text: one-pager example).
+- The sections are created using (link: https://github.com/TimOetting/kirby-builder text: Tim Oetting's Builder plugin).
+
+Both approaches have their pros and cons.
+
+### Subpages
+
+- Subpages can be as complex as needed and can themselves contain structure fields or subpages if needed.
+- Subpages are probably easier to handle if the sections contain a lot of content.
+- Supages are by default accessible via their URL.
+- Without a plugin that allows you to skip the title, every subpage needs a title.
+
+Options to circumvent some disadvantages:
+
+- Store sections in their own sections subfolder.
+- Create (link: text: page models) to rewrite the Urls.
+- Use (link: docs/guide/routing text: routes) to prevent access to the section URLs.
+- Use the (link: https://github.com/steirico/kirby-plugin-custom-add-fields text: Custom Add Fields plugin) that allows skipping the title when creating new section pages.
+
+
+### The Builder plugin
+
+- All content is created in the same page.
+- Nested structures are possible but should probably be limited to two levels. Nested content cannot be moved between blocks.
+- Easy creation of new blocks.
+- Better preview of final result.
+- No redirecting necessary.
+- Currently not compatible with the (link: https://github.com/getkirby/editor text: Editor plugin)
+
+
+## The media folder
+
+Whenever you upload a file to a page via the Panel, the file is uploaded to the content folder. However, as soon as the URL of the file is called in the browser, the file is moved to the media folder. Also, all thumbnails are created on the fly and stored in the media folder. Therefore, this folder needs to be publicly accessible within the document root. While this setup needs more space on the server, it has some real advantages, like providing consistent image URLs or being able to move the content folder (and most other folders) above the document root for better security (see [custom folder setup](/docs/guide/configuration#custom-folder-setup)).
```

Comment:
Files are not moved to the `media` folder but copied.

Reply:
So true